app/src/main/java/org/p2p/wallet/common/feature_toggles/remote_config/AppFirebaseRemoteConfig.kt

```diff
@@ -106,6 +106,13 @@ class AppFirebaseRemoteConfig(
         return remoteConfig.getString(toggleKey).toIntOrNull()
     }
 
+    override fun getLong(toggleKey: String): Long? {
```

Comment:
Perhaps for clarity, all these `get*` functions should be separated into two distinct ones:
1. `getLongOrNull(): Long?`
2. `getLong(key, default: Long): Long`

Reply:
it's a inside impl of FirebaseRemoteConfig
we pass default values inside the feature toggles, so i think there no need for multiple methods